programs/programs/nc/medicaid/emergency_medicaid/calculator.py

```diff
@@ -0,0 +1,42 @@
+from programs.programs.calc import MemberEligibility, ProgramCalculator, Eligibility
+from programs.programs.helpers import medicaid_eligible
+import programs.programs.messages as messages
+
+
+class EmergencyMedicaid(ProgramCalculator):
+    # $6,268/yr | ~$522/mo
+    member_amount = 6268
+    max_age = 64
+    fpl_percent = 1.96
+    dependencies = ["age", "insurance", "income_amount", "household_size"]
+
+    def household_eligible(self, e: Eligibility):
+        # Does not have insurance
+        has_no_insurance = False
+        for member in self.screen.household_members.all():
+            has_no_insurance = member.insurance.has_insurance_types(("none",)) or has_no_insurance
+
+            # Pregnant and under 18 years old have a different FPL percentage
+            if member.age <= 18 and member.pregnant:
+                EmergencyMedicaid.fpl_percent = 2.11
```

Comment:
This is permanently changing the `fpl_percent` on this class.

Reply:
good catch, thanks